packages/genesys-spark-components/src/components/stable/gux-popover/gux-popover.scss

```diff
@@ -29,12 +27,7 @@
   .gux-arrow-caret {
     inline-size: 0;
     block-size: 0;
-    filter: drop-shadow(
-      var(--gse-ui-popover-boxShadow-offsetX)
-        var(--gse-ui-popover-boxShadow-offsetY)
-        calc(var(--gse-ui-popover-boxShadow-blur) / 2)
-        var(--gse-ui-popover-boxShadow-color)
-    );
+    filter: drop-shadow(0 0 4px var(--gse-semantic-effects-boxShadow));
```

Comment:
Token for 4px?

Reply:
I don't think so I think this is a technical issue with our caret implementation.